source/layouts/guide.erb

```diff
@@ -1,10 +1,14 @@
 <!DOCTYPE html>
 <html lang="en">
   <%
+    require 'json'
+    require 'open-uri'
+
+    @version = JSON.parse(open('https://guides.emberjs.com/versions.json').read).last
```

Comment:
@locks Instead of reading this from a web page, couldn't we just read it from the local file on build?